storage/Devices/BcacheImpl.cc

```diff
@@ -144,6 +161,54 @@ namespace storage
     }
 
 
+    static CacheMode
+    parse_mode(File cache_mode_file)
+    {
+        string modes = cache_mode_file.get<string>();
+        regex rgx("\\[(.*)\\]");
+        smatch match;
+        if ( regex_search(modes, match, rgx) )
+        {
+
+            return toValueWithFallback(match[1], CacheMode::WRITETHROUGH);
+        }
+        else
+        {
+            y2err("Unknown value in cache_mode '" << modes << "'");
```

Comment:
This is misleading. This occurs for an "unparseable file", while an unknown value, like "writefancypants" is *silently* converted by `toValueWithFallback` above, right?

Reply:
yeah, result of previous alghorithm

Reply:
I don't understand. Anyway:
- change the error message to "Cannot parse cache_mode '"
- raise an exception, both for unparseable input, and a new unknown enum value, which is consistent with the rest of the library, I HOPE.

Reply:
I check it and majority of library use variant with fallback. It will log error for failing case. So I keep it as it is.